app/src/main/java/org/resolvetosavelives/red/RedApp.kt

```diff
@@ -1,13 +1,17 @@
 package org.resolvetosavelives.red
 
 import android.app.Application
+import io.sentry.Sentry
+import io.sentry.android.AndroidSentryClientFactory
 import org.resolvetosavelives.red.di.AppComponent
 import org.resolvetosavelives.red.di.AppModule
 import org.resolvetosavelives.red.di.DaggerAppComponent
 import timber.log.Timber
 
 class RedApp : Application() {
 
+  val dsn = "https://e8606e0cd2f7470e834e446905612ff2@sentry.io/1212614"
```

Comment:
Can you rename this to something other than `dsn`? Sounds very vague. Maybe you can also make this a constant by moving this to a companion object:

```kotlin
companion object {
  const val DSN = "…"
}
```

I think this is the java equivalent of static fields.

Reply:
Is this `dsn` safe to include in our repo publicly?